src/utils/log.js

```diff
@@ -46,13 +46,16 @@ const banner = `
 | | | / __| | '_ \\ / _\` |   | |/ _ \\ '_ \` _ \\| '_ \\| |/ _\` | __/ _ \\
 | |_| \\__ \\ | | | | (_| |   | |  __/ | | | | | |_) | | (_| | ||  __/
  \\___/|___/_|_| |_|\\__, |   |_|\\___|_| |_| |_| .__/|_|\\__,_|\\__\\___|
-                   |___/                     |_|              v${pack.version}
-`;
+                   |___/                     |_|              v${pack.version}`;
 
-const goToStep = (step) => {
+const goToStep = (step, templateBanner = undefined) => {
   if (stepBanners[step - 1] !== undefined) {
     console.log(ansi.erase.screen);
     console.log(banner);
+    if (templateBanner) {
```

Comment:
Should we still show the `Using Template` banner if the template provides a banner ? Maybe use a smaller banner if one is provided so its not overpowering

Reply:
yeah i like that.

Reply:
Agreed

Reply:
Addressed in a4cdfb1 
Just kept the version as thats the only useful piece of information
![image](https://user-images.githubusercontent.com/2575450/134028447-f582bb47-867e-47d0-9a0a-bc544d485d89.png)

Reply:
Addressed in a4cdfb1

Just kept the version as this is the only useful piece of information